proxy/src/serverless/conn_pool_lib.rs

```diff
@@ -153,9 +233,11 @@ impl<C: ClientInnerExt> EndpointConnPool<C> {
 
         // do logging outside of the mutex
         if returned {
-            info!(%conn_id, "pool: returning connection '{conn_info}' back to the pool, total_conns={total_conns}, for this (db, user)={per_db_size}");
+            info!(%conn_id, "{}: returning connection '{conn_info}' back to the pool, total_conns={total_conns},
+                                 for this (db, user)={per_db_size}", pool_name);
```

Comment:
Newlines don't render well in our logs
```suggestion
            info!(%conn_id, "{pool_name}: returning connection '{conn_info}' back to the pool, total_conns={total_conns}, for this (db, user)={per_db_size}");
```